couler/core/templates/job.py

```diff
@@ -37,7 +37,7 @@ def __init__(
         self.args = args
         self.action = action
         self.manifest = manifest
-        self.set_owner_reference = "true" if set_owner_reference else "false"
+        self.set_owner_reference = utils.bool_to_str(set_owner_reference)
```

Comment:
Could you check if there are other places that require this and update those as well?

Reply:
this looks to be the only one place in the code base.

Reply:
Thanks for checking. It can probably be reused in our internal extensions.